src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -123,46 +120,5 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             });
 //[#endif]
         }
-#if CertificateAuth
```

Comment:
Always fun to remove 🎉

Reply:
Yes it is! It's also fun to open a PR without running the tests so 🤞 😂

Reply:
Too bad, would've been too easy 😂